### ISSUE TITLE
Win bundle arduino-cli not found

### ISSUE THREAD

**Issue by smoluks** [vscode-arduino/vscode-arduino#27]
Steps to reproduce:

1. Install plugin (v0.7.0)
useArduinoCli set up to true by default:
arduino.path and arduino.commandPath clear by default

2. See console:
[Error] Analyzing sketch 'firmware.ino': {"error":{"errno":-4058,"code":"ENOENT","syscall":"spawn c:\\Users\\Administrator\\.vscode\\extensions\\vscode-arduino.vscode-arduino-community-0.7.0-win32-x64\\assets\\platform\\win32-x64\\arduino-cli\\arduino-cli.exe","path":"c:\\Users\\Administrator\\.vscode\\extensions\\vscode-arduino.vscode-arduino-community-0.7.0-win32-x64\\assets\\platform\\win32-x64\\arduino-cli\\arduino-cli.exe","spawnargs":["compile","-b","esp32:esp32:esp32:JTAGAdapter=default,PSRAM=disabled,PartitionScheme=app3M_fat9M_16MB,CPUFreq=240,FlashMode=qio,FlashFreq=80,FlashSize=16M,UploadSpeed=921600,LoopCore=0,EventsCore=0,DebugLevel=info,EraseFlash=none,ZigbeeMode=default","--verbose","--no-color","--build-path","c:\\Users\\Administrator\\Desktop\\clock_esp\\firmware\\build","c:\\Users\\Administrator\\Desktop\\clock_esp\\firmware\\firmware.ino","--additional-urls",""]}}

Win files are missing in the directory, only darwin:
![{B802CF1B-24F3-4985-86E4-2BD60B0C7485}](https://github.com/user-attachments/assets/b4d2230a-a8ad-48ab-96cf-f70809ef4fe0)

**Comment by lersi** [comment]
Okay, managed to narrow down the problem, it was a compilation mistake that included the wrong set of tools, sorry I will fix it ASAP

**Comment by lersi** [comment]
Please let me know if this issue persists in release 0.7.1

**Comment by smoluks** [comment]
Yep
![{1A379356-E172-46F2-87AF-0CF5779586E3}](https://github.com/user-attachments/assets/fb31e110-1a16-47e7-899b-96e3ded7ce4b)
![{D8836E29-7FAC-4E07-9566-B98566FEC878}](https://github.com/user-attachments/assets/721d13cc-1e01-429d-a1c7-cf4bf2de1720)

**Comment by vicatcu** [comment]
Yes, still a problem. Had to re-do the fix when something automatically updated today when I opened VSCode.

**Comment by vicatcu** [comment]
When I try and open from the command palette: "Arduino: Board Config" I get an error now (v0.7.1):
```
Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state..
```

**Comment by lersi** [comment]
Please let me know if this issue persists in version "0.7.2" (dummy version)

**Comment by vicatcu** [comment]
0.7.2 update didn't cause any trouble. Works as though I had manually patched the previous version.

**Comment by lersi** [comment]
This problem was caused by vsce tool, closing this issue as solve
Thanks for the info and the help
